packages/jaeger-ui/src/model/trace-dag/TraceDag.js

```diff
@@ -106,6 +106,7 @@ export default class TraceDag<T = void> {
     if (!skipToChild) {
       const node = this._getDagNode(service, operation, children.size > 0, parentNodeID, data);
       node.count++;
+      node.members.push(denseSpan);
```

Comment:
Instead of having the `TraceDag` mutate a property on the `DagNode`, it would be cleaner if the `DagNode` class had a function to update `this.members`, then `TraceDag` could call `node.addMember(denseSpan)`

Reply:
Good idea. Looks cleaner.